package.json

```
@@ -13,12 +13,10 @@
     "cy:install": "node ./node_modules/cypress/bin/cypress install",
     "cy:open": "cypress open --env env=watch",
     "cy:run": "cypress run --env env=watch",
-    "test:watch": "start-server-and-test playground:dev http://localhost:8000 'cypress open --env env=watch'",
-    "test:admindev": "start-server-and-test playground:dev http://localhost:1337 'cypress run --env env=watch'",
-    "test:dev": "start-server-and-test playground:soft_dev http://localhost:1337 'cypress run --env env=develop'",
-    "test:production": "start-server-and-test playground:production http://localhost:1337 'cypress run --env env=test'",
     "pretest": "yarn playground:build",
-    "test": "start-server-and-test playground:start http://localhost:1337 'cypress run --env env=test'"
+    "test": "concurrently \"NODE_ENV=test yarn playground:start\" \"cypress run --env env=test\"",
+    "test:watch": "concurrently \"NODE_ENV=test yarn playground:dev\" \"cypress open --env env=watch\"",
+    "test:production": "concurrently \"NODE_ENV=production yarn playground:production\" \"cypress run --env env=test\""
```

```suggestion
    "test": "concurrently --kill-others -s first \"NODE_ENV=test yarn playground:start\" \"cypress run --env env=test\"",
    "test:watch": "concurrently --kill-others -s first \"NODE_ENV=test yarn playground:dev\" \"cypress open --env env=watch\"",
    "test:production": "concurrently --kill-others -s first \"NODE_ENV=production yarn playground:production\" \"cypress run --env env=test\""
```

`--kill-others` is required so that strapi is shutdown after the tests are completed
`--s first` is required for reporting the of cypress if it is the first to complete or strapi if it fails before that